tests/test_google_sheets_all_fields.py

```diff
@@ -80,7 +80,10 @@ def test_run(self):
                 self.assertGreater(len(expected_all_keys), len(expected_automatic_keys))
                 self.assertTrue(expected_automatic_keys.issubset(expected_all_keys), msg=f'{expected_automatic_keys-expected_all_keys} is not in "expected_all_keys"')
                 if stream == "file_metadata":
-                    #  BUG | below keys are not synced https://jira.talendforge.org/browse/TDL-14409
+
+                    # As per google documentation https://developers.google.com/drive/api/v3/reference/files `teamDriveId` is deprecated. There is mentioned that use `driveId` instead.
```

Comment:
If these streams don't exist we should not be adding them in discovery. The `expected_all_keys` is set based on the catalog produced by the tap in discovery mode. So if these fields are not returned by the google api, we must explicitly be adding them in the tap. This is still a bug, but a different one.

Reply:
Agree @kspeer825  we should not be adding `teamDriveId` and `driveId` fields in the discovery. There will be one behavior change in which the user will not be able to see these fields in UI.
Should we remove these fields from the schema file https://github.com/singer-io/tap-google-sheets/blob/master/tap_google_sheets/schemas/file_metadata.json#L22-L27?
CC: @KrisPersonal